crates/red_knot_python_semantic/resources/mdtest/comparison/tuples.md

```diff
@@ -58,7 +58,22 @@ reveal_type(c >= d)  # revealed: Literal[True]
 #### Results with Ambiguity
 
 ```py
-def _(x: bool, y: int):
+class P:
+    def __lt__(self, other: "P") -> bool:
+        return True
+
+    def __le__(self, other: "P") -> bool:
+        return True
+
+    def __gt__(self, other: "P") -> bool:
+        return True
+
+    def __ge__(self, other: "P") -> bool:
+        return True
+
+class Q(P): ...
+
+def _(x: P, y: Q):
```

Comment:
I changed this because it doesn't look like we've implemented comparisons between unions inside tuples yet, and `bool` is now internally represented as a union, which meant that many `reveal_type` assertions in this test became TODOs rather than `bool`s. Testing whether we knew how to compare a `bool` with a `bool` didn't seem to me to be the point of the test.